docs/en/setup/backend/backend-aws-eks-monitoring.md

```diff
@@ -0,0 +1,265 @@
+# AWS Cloud EKS monitoring
```

Comment:
@pg-yang I updated the doc a little. Mostly because you put an empty space between the last word and the comma. This is not correct from English grammar perspective.

Reply:
Thanks, I will be more careful with this mistake.